draft-secure-credential-transfer.md

```diff
@@ -114,7 +114,7 @@ API parameters:
 
 # Credential transfer workflows
 
-We define two flows for credential transfer: 1. Stateless (Relay server facilitates a single credential data transfer: Sender -> Relay -> Receiver) and 2. Stateful (Relay server facilitates additional data transfers - there are multiple data transfers in this flow to prepare credential data for registering or provisioning by Receiver). The details are provided below.
+We define two flows for credential transfer: 1. Stateless (Relay server facilitates a single credential data transfer: Sender -> Relay -> Receiver) and 2. Stateful (Relay server facilitates additional data transfers - there are multiple data transfers in this flow to prepare credential data for registering or provisioning by Receiver). Relay server does not limit the number of such data tranfsfers between Sender and Receiver devices. The details are provided below.
```

Comment:
While the Relay server doesn't limit the number of round trips we've said that reusing the same symmetric key isn't sufficient from a security POV. I don't know if we need to include that here, but wanted to call out that if we were to use this for a chatty provisioning we might need to use the first few messages to upgrade the channel's encryption.